perseo-main/src/main/java/com/telefonica/iot/perseo/Utils.java

```diff
@@ -76,12 +78,31 @@ public static synchronized EPServiceProvider initEPService(ServletContext sc) {
             cfg.addEventType(Constants.IOT_EVENT, def);
 
             // Add SunriseSunset library
-            cfg.addImport("ca.rmen.sunrisesunset.*");
+            //cfg.addImport("ca.rmen.sunrisesunset.*");
```

Comment:
Leftover comment?

Reply:
Fixed in [f3f840b](https://github.com/telefonicaid/perseo-core/pull/143/commits/f3f840b4bcf70c19c14266beabc9f568ec1feec1)